src/Api/Auth/Controllers/WebAuthnController.cs

```diff
@@ -93,6 +114,29 @@ private async Task ValidateRequireSsoPolicyDisabledOrNotApplicable(Guid userId)
         }
     }
 
+    [HttpPut()]
+    public async Task UpdateCredential([FromBody] WebAuthnLoginCredentialUpdateRequestModel model)
+    {
+        var tokenable = _assertionOptionsDataProtector.Unprotect(model.Token);
+        if (!tokenable.TokenIsValid(WebAuthnLoginAssertionOptionsScope.PrfRegistration))
+        {
+            throw new BadRequestException("The token associated with your request is expired. A valid token is required to continue.");
```

Comment:
nit(non-blocking): Technically they could've sent a token with another scope

```suggestion
            throw new BadRequestException("The token associated with your request is invalid. A valid token is required to continue.");
```

```suggestion
            throw new BadRequestException("The token associated with your request is invalid or has expired. A valid token is required to continue.");
```